CreateTeamsalongwithAPPs/CreateTeamsAlongWithApps.ps1

```diff
@@ -0,0 +1,126 @@
+
+$logfile = "C:\log_$(get-date -format `"yyyyMMdd_hhmmsstt`").txt"
```

Comment:
similar suggestion on logging, rather be more verbose in logging for long scripts as it helps admins to understand what happens and where things fail.